### DISCUSSION TITLE
How do i uninstall onedrive?

### DISCUSSION THREAD

**Discussion by ghost** [abraunegg/onedrive#2923] [Q&A] [answered]
i am not tech-addicted but i like ubuntu (currently using 24.04 LTS) for usability und seceurity.

since i finished university and my microsoft onedrive subscription is running out i would like to completely uninstall onedrive and delete all the dependencies and settings i changed following the official instructions by abraunegg. like as if it has never been installed.

what commands do i have to use in the terminal?

thanks in advance! :)

**Comment by abraunegg** [comment]
@toblmops 
My first question would be - what instructions did you follow to install?

**Reply by ghost** [reply]
first i installed it following the instructions on wiki.ubuntuusers (couple of years ago) but then i uninstalled and reinstalled it following your instructions on https://github.com/abraunegg/onedrive/blob/master/docs/ubuntu-package-install.md

**Comment by abraunegg** [comment, marked as answer]
@toblmops 
As you are using Ubuntu, then as per your distributions instructions:

```
sudo apt remove onedrive
```

**Reply by ghost** [reply]
oh okay - simple as that!
so this single command removes the repositorys and all the changes combined with onedrive as well?

**Reply by abraunegg** [reply]
If you use `onedrive --display-config` the output will show you where any configuration files are which you can remove.

Beyond that, I suggest you seek support from Ubuntu for anything else.
